data/actions/scripts/other/potions.lua

```diff
@@ -27,132 +72,90 @@ function onUse(player, item, fromPosition, target, toPosition, isHotkey)
 	if target == nil or not target:isPlayer() then
 		return true
 	end
-
+	
 	if player:getCondition(CONDITION_EXHAUST_HEAL) then
 		player:sendTextMessage(MESSAGE_STATUS_SMALL, Game.getReturnMessage(RETURNVALUE_YOUAREEXHAUSTED))
 		return true
 	end
 
-	local itemId = item:getId()
-	if itemId == antidotePot then
-		if not antidote:execute(target, numberToVariant(target:getId())) then
-			return false
-		end
-
+	local stats = potions[item:getId()]
```

Comment:
A better name for this could be: **potionId**, **isPotion** or just **potion**.
I mean it is easier to understand what **stats** mean by looking at the right side of its assignment.